charts/kubernetes-sidecar-injector/values.yaml

```diff
@@ -18,6 +18,11 @@ podAnnotations: {}
 podLabels: {}
 podSecurityContext: {}
 
+pdb:
+  enabled: false
```

Comment:
id prefer this be `create` instead of `enabled`

Reply:
I'll change this